src/ground_truth_nsrts.py

```diff
@@ -33,7 +35,7 @@ def get_gt_nsrts(predicates: Set[Predicate],
         nsrts = _get_blocks_gt_nsrts()
     elif CFG.env == "behavior":
         nsrts = _get_behavior_gt_nsrts()  # pragma: no cover
-    elif CFG.env == "painting":
+    elif CFG.env in ("painting", "repeated_nextto_painting"):
```

Comment:
while we're at it, just for consistency can you change the above line:
`elif CFG.env in ["blocks", "pybullet_blocks"]:`
to:
`elif CFG.env in ("blocks", "pybullet_blocks"):`
sorry and thanks!